ember-core/src/main/scala/org/http4s/ember/core/Parser.scala

```diff
@@ -103,7 +103,7 @@ private[ember] object Parser {
         bv: Array[Byte],
         accHeaders: List[v2.Header.Raw],
         idx: Int,
-        state: Byte,
+        state: Boolean,
```

Comment:
A "state" of boolean seems like a big smell to me, but we can sort that out later.